pages/instructorQuestionManualGrading/instructorQuestionManualGrading.js

```diff
@@ -0,0 +1,104 @@
+            sqlDb.callZeroOrOneRow('instance_question_select_manual_grading_objects', params, (err, result) => {
+                if (ERR(err, next)) return;
+
+                // Instance question doesn't exist (redirect to config page)
+                if (result.rowCount == 0) {
+                    return callback(error.make(404, 'Instance question not found.', {locals: res.locals, body: req.body}));
+                }
+
+                /**
+                 * Student never loaded question (variant and submission is null)
+                 * Student loaded question but did not submit anything (submission is null)
+                 */
+                if (!result.rows[0].variant || !result.rows[0].submission) {
+                    return callback(error.make(404, 'No gradable submissions found.', {locals: res.locals, body: req.body}));
+                }
+
+                res.locals.question = result.rows[0].question;
+                res.locals.variant = result.rows[0].variant;
+                res.locals.submission = result.rows[0].submission;
+                res.locals.score_perc = res.locals.submission.score * 100;
+                callback(null);
+            });
+        },
+       (callback) => {
+            res.locals.overlayGradingInterface = true;
+            question.getAndRenderVariant(res.locals.variant.id, null, res.locals, function (err) {
+              if (ERR(err, next)) return;
+              callback(null);
+            });
+        },
+    ], (err) => {
+        if (ERR(err, next)) return;   
+        res.render(__filename.replace(/\.js$/, '.ejs'), res.locals);
+    });
+
+    debug('GET /');
+});
+
+router.post('/', function(req, res, next) {
+    if (req.body.__action == 'add_manual_grade') {
+        const note = req.body.submission_note;
+        const score = req.body.submission_score;
+        const params = [res.locals.instance_question.id];
+
+        sqlDb.callZeroOrOneRow('instance_question_select_manual_grading_objects', params, (err, result) => {
+            if (ERR(err, next)) return;
+
+            const {question, variant, submission} = result.rows[0];
+            if (!question || !variant || !submission) return next(error.make('500', 'Manual grading dependencies missing'));
+
+            Object.assign(res.locals, {question, variant, submission});
+
+            const params = [
+                submission.id,
+                res.locals.authn_user.user_id,
+                submission.gradable,
+                submission.broken,
+                submission.format_errors,
+                submission.partial_scores,
+                score / 100, // overwrite submission score
+                submission.v2_score,
+                {manual:note}, // overwrite feedback
+                submission.submitted_answer,
+                submission.params,
+                submission.true_answer,
+            ];
+
+            sqlDb.callOneRow('grading_jobs_insert_internal', params, (err, result) => {
```

Comment:
I think we should make a new `grading_jobs_insert_manual` for use here.

This should be similar to the existing `grading_jobs_insert_external_manual` and `grading_jobs_insert_internal`.

We should rename `grading_jobs_insert_external_manual` to just `grading_jobs_insert_external`.